CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Services/ApiClient.cs

```diff
@@ -79,5 +79,41 @@ public async Task<HttpResponseMessage> PostAsync(string resource, string body)
                 throw;
             }
         }
+        public async Task<HttpResponseMessage> PutAsync(string resource, string body)
```

Comment:
Separate with an empty line. Rename method to **UpdateAsync**, when giving a name for the method we should be platform, architecture agnostic, _Put_ is convention for REST API, it should affect us when writing code as a client, we only use it this name when provided by external libraries